cmd/agent/app/reporter/grpc/flags.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpc
+
+import (
+	"flag"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	gRPCPrefix        = "reporter.grpc."
+	collectorHostPort = "collector.host-port"
```

Comment:
grpc dial supports only one address by default. An option is to use with balancer https://stackoverflow.com/a/50302295/4158442 not sure what that implies though.

Reply:
the load balancer implementation can be done later, I'm just asking whether the cmd line should support a list of addresses (it can fail for now on >1 address).